pkg/datapath/linux/node.go

```diff
@@ -522,6 +525,55 @@ func (n *linuxNodeHandler) encryptNode(newNode *node.Node) {
 
 }
 
+func neighborLog(spec string, err error, ip *net.IP, hwAddr *net.HardwareAddr, link int) {
+	scopedLog := log.WithFields(logrus.Fields{
+		logfields.Reason: spec,
+		"IP":             ip,
+		"HardwareAddr":   hwAddr,
+		"LinkIndex":      link,
+	})
+
+	if err != nil {
+		scopedLog.WithError(err).Error("insertNeighbor failed")
+	} else {
+		scopedLog.Info("insertNeighbor")
```

Comment:
Do we want to be printing info messages every time like this? In large scale clusters this will flood the logs, no?

Reply:
no lets dropping it.

Reply:
OK, I decided to leave it but made it Debug.